src/OrchardCore.Modules/OrchardCore.Flows/Views/ContentCard-FlowPart.Edit.cshtml

```diff
@@ -39,17 +39,19 @@
                         <ul class="dropdown-menu dropdown-menu-end">
                             @foreach (var type in widgetContentTypes.OrderBy(w => w.DisplayName))
                             {
-                                <li><a class="dropdown-item insert-widget btn-sm"
-                                   data-target-id="@Model.TargetId"
-                                   data-html-field-prefix="@Model.HtmlFieldPrefix"
-                                   data-prefixes-name="@Model.PrefixesName"
-                                   data-flowmetadata="true"
-                                   data-parent-content-type="@Model.ParentContentType"
-                                   data-part-name="@Model.CollectionPartName"
-                                   data-contenttypes-name="@Model.ContentTypesName"
-                                   data-contentitems-name="@Model.ContentItemsName"
-                                   data-widget-type="@type.Name"
-                                   href="javascript:;">@type.DisplayName</a></li>
```

Comment:
I assume that you used VS auto-formatting here. Maybe that's not the best.

Reply:
@Skrypt updated!